### PR TITLE
feat: add email providers

### PR DESCRIPTION
**Pull request by ndom91**
- Add email providers to provider data json file under their own `providersEmail` key.
- Rename oauth providers key to `providersOAuth` (incl. updated everywhere where it was being used)
- Move popular providers to top of that list, so they get shown first
	- I know, probably not going to be useful once Lluis's dynamic provider PR ships, but who knows where else we might use this list as-is across the docs
- Add svg logos for
	1. Nodemailer
	2. Resend
	3. Sendgrid

Snuck in these two tiny navbar changes :blush: :
- Fix navbar link for `/getting-started`
- Fix navbar dynamic loading / layout shift for AI button

image

### CODE REVIEW

File: pages/data/manifest.json [hunk 1]
```diff
@@ -42,11 +42,17 @@
...
-    "42-school": "42 School",
+    "github": "GitHub",
+    "google": "Google",
+    "twitter": "Twitter",
     "apple": "Apple",
+    "discord": "Discord",
+    "auth0": "Auth0",
+    "resend": "Resend",
+    "sendgrid": "Sendgrid",
+    "42-school": "42 School",
```
Comment (0ubbe) [inline pages/data/manifest.json:51]: I would keep the email providers separate from the OAuth ones 💅🏽 

For email, I guess a database does need to be set right ( _to store verification tokens..._ ), so the instructions can't be the same as OAuth and I wouldn't group them together.

Suggest we rename this object to:
```json
"providers-oauth": {
  "github": "Github",
   ...
 }, 
 "providers-email": {
  "resend": "Resend",
   ...
 }, 
 ```

Comment (ndom91) [reply pages/data/manifest.json:51]: Yeah good idea. Pushed!

**Review by balazsorban44 — CHANGES_REQUESTED** [review]
Nodemailer, Resend and SendGrid should not show up in the OAuth tab, but on Token auth one.

We should the same combobox as on the OAuth tab, as we plan on adding more providers like this in the future

File: components/LogosMarquee/Slider.tsx [hunk 2]
```diff
@@ -47,7 +47,7 @@ export const ProviderMarquee = memo(() => {
         scatterRandomly
         resetAfterTries={200}
       >
-        {Object.entries(manifest.providers)
+        {Object.entries(manifest.providersOAuth)
```
Comment (balazsorban44) [inline components/LogosMarquee/Slider.tsx:50]: Let's showcase the email providers too!
```suggestion
        {Object.entries({ ...manifest.providersOAuth, ...manifest.providersEmail })
```

File: components/LogosMarquee/index.tsx [hunk 3]
```diff
@@ -14,7 +14,7 @@ export function LogosMarquee() {
         gradientColor="#171717"
         autoFill
       >
-        {Object.entries(manifest.providers)
+        {Object.entries(manifest.providersOAuth)
```
Comment (balazsorban44) [inline components/LogosMarquee/index.tsx]:
```suggestion
        {Object.entries({ ...manifest.providersOAuth, ...manifest.providersEmail })
```

**Comment by ndom91** [comment]
Add EmialProviders to marquee on homepage, but they're not rendering for some reason :thinking:

**Review by balazsorban44 — APPROVED** [review]

**Pull request merged**